backend/src/libs/sequelize.js

```diff
@@ -1,7 +1,7 @@
 const Sequelize = require('sequelize');
 const log         = require('./log')(module);
 
-const sequelize = new Sequelize('postgres://vagrant:vagrant@127.0.0.1:5432/pkeeper');
+const sequelize = new Sequelize('postgres://vagrant:vagrant@postgres:5432/pkeeper');
```

Comment:
I thinking about it and not found the decision, how to keep both option.
I think, i will delete everything, related to vagrant.